cmake/Corrosion.cmake

```diff
@@ -436,71 +443,73 @@ function(_corrosion_add_library_target)
     endif()
     set("${CALT_OUT_ARCHIVE_OUTPUT_BYPRODUCTS}" "${archive_output_byproducts}" PARENT_SCOPE)
 
-    add_library(${target_name} INTERFACE)
+    add_library(${target_name_cmake} INTERFACE IMPORTED GLOBAL)
```

Comment:
Is there a specific reason why you added `IMPORTED GLOBAL` here? I'm pretty sure it _probably_ doesn't change anything, but then again this is the kind of change that might subtly break something some user is relying on. I would prefer if this change was removed from this PR. If you would like to add `IMPORTED_GLOBAL`, then please open a separate PR with just this change.